lib/anyplayer/players/itunes_windows.rb

```diff
@@ -63,11 +63,9 @@ def platforms
     [:windows]
   end
 
-
   private
 
   def itunes
-    itunes ||= WIN32OLE.new("iTunes.Application")
+    itunes || WIN32OLE.new("iTunes.Application")
```

Comment:
This change looks a bit weird. I'm afraid this will loop infinitely under Windows. Are we sure this is safe?